dough-data/src/main/java/io/github/thebusybiscuit/dough/collections/CopyUtils.java

```diff
@@ -189,10 +189,10 @@ private CopyUtils() {}
      *            The type of elements in the arrays
      *
      */
-    public static <T> @Nonnull T[] deepCopy(@Nonnull T[] source, @Nonnull UnaryOperator<T> cloningFunction, @Nonnull IntFunction<T[]> sinkSupplier) {
+    public static @Nonnull <T> T[] deepCopy(@Nonnull T[] source, @Nonnull UnaryOperator<T> cloningFunction, @Nonnull IntFunction<T[]> sinkSupplier) {
```

Comment:
Why were these all moved? The annotation applies to the return type `T[]`

Reply:
It won't compile if I don't move them.

Reply:
*rolls eyes*